src/ui/ons/ONSCollapsible.svelte

```diff
@@ -62,7 +62,7 @@
       aria-hidden="true"
     >
       <span class="ons-btn__inner ons-js-collapsible-button-inner">Hide this</span>
-      <span class="ons-btn__context ons-u-vh">What is a photovoltaic system? content</span>
+      <span class="ons-btn__context ons-u-vh">{title ? title : ""}</span>
```

Comment:
Wouldn't it be better to hide the whole `ons-btn__context` element if `title` is missing?

Reply:
Good shout, it seems this check actually doesn't even need to be here anyway as further up the file we are already defaulting title to an empty string if not present. I could wrap it in an if statement but this component is only used within ExploreByTopic and we have hardcoded the prop `<ONSCollapsible title="Show me suggestions" a11yHeading>` so maybe we need no check there at all at the moment, although the if statement is better for future-proofing. Let me know what you think.

Reply:
I see. Just checked with https://ons-design-system.netlify.app/components/collapsible/ and the `title` should be required therefore the default empty string shouldn't be there in the first place.

I know this is slightly beyond the scope of this PR but since we're improving it, may I suggest:
- dropping the default value so it's just `export let title` 
- getting rid of all the ternaries now that the `title` is required